core/lib/config/src/configs/fri_witness_generator.rs

```diff
@@ -51,15 +53,20 @@ impl WitnessGenerationTimeouts {
         self.node
     }
 
+    pub fn recursion_tip(&self) -> Duration {
+        self.recursion_tip
+    }
+
     pub fn scheduler(&self) -> Duration {
         self.scheduler
     }
 
-    pub fn new(basic: u16, leaf: u16, node: u16, scheduler: u16) -> Self {
+    pub fn new(basic: u16, leaf: u16, node: u16, recursion_tip: u16, scheduler: u16) -> Self {
         Self {
             basic: Duration::from_secs(basic as u64),
```

Comment:
Nit: Since `u16` -> `u64` conversion is lossless, [it's more idiomatic](https://rust-lang.github.io/rust-clippy/master/index.html#/cast_lossless) to use `From` / `Into` traits for it to signal it explicitly, e.g. `basic.into()`.